bin/control.sh

```diff
@@ -42,6 +42,9 @@ export AKKA_CONF=${AKKA_CONF:-"application.conf"}
 export ALERT_ENGINE_MAX_HEAP=${ALERT_ENGINE_MAX_HEAP:-"2G"}
 export COLLECTION_ROLLER_MAX_HEAP=${COLLECTION_ROLLER_MAX_HEAP:-"2G"}
 export LOG_COLLECTOR_MAX_HEAP=${LOG_COLLECTOR_MAX_HEAP:-"8G"}
+export KAFKA_PROPS=${KAFKA_PROPS:-"./cloudera-integration/csd/aux/kafka.properties"}
```

Comment:
This is correct when you run it locally but will fail when deployed on a cluster. The correct default value on a cluster should be just `kafka.properties`